src/channel/playlist.js

```diff
@@ -594,7 +594,7 @@ PlaylistModule.prototype.handleDelete = function (user, data) {
     var plitem = this.items.find(data);
     self.channel.refCounter.ref("PlaylistModule::handleDelete");
     this.semaphore.queue(function (lock) {
-        if (self._delete(data)) {
+        if (self._delete(data, user.getName())) {
```

Comment:
`getName()` could be null for anonymous users.  I think the voteskip log coalesces this to `(anonymous)`.

This actually makes me realize I think the log statement immediately below this is also flawed and will log "null deleted blahblah".